src/moltin.d.ts

```diff
@@ -1,32 +1,31 @@
 // Type definitions for @moltin/js-sdk
 // Project: @moltin/sdk
-
-import * as config from './types/config'
-import * as storage from './types/storage'
-import * as product from './types/product'
-import * as core from './types/core'
-import * as customer from './types/customer'
-import * as order from './types/order'
-import * as cart from './types/cart'
-import * as address from './types/address'
-import * as inventory from './types/inventory'
-import * as field from './types/field'
-import * as collection from './types/collection'
-import * as category from './types/category'
-import * as brand from './types/brand'
-import * as currency from './types/currency'
-import * as integration from './types/integration'
-import * as job from './types/job'
-import * as file from './types/file'
-import * as flow from './types/flow'
-import * as transaction from './types/transaction'
-import * as settings from './types/settings'
-import * as promotions from './types/promotions'
-import * as variations from './types/variations'
-import * as authenticationSettings from './types/authentication-settings'
-import * as oidcProfile from './types/oidc-profile'
-import * as authenticationRealm from './types/authentication-realm'
-import * as paymentGateway from './types/gateway'
+import { AuthenticateResponseBody, Config, ConfigOptions, RequestFactory } from './types/config'
+import { StorageFactory } from './types/storage'
+import { ProductsEndpoint } from './types/product'
+import { PcmProductsEndpoint } from './types/pcm'
+import { CurrencyEndpoint } from './types/currency'
+import { BrandEndpoint } from './types/brand'
+import { CategoryEndpoint } from './types/category'
+import { CollectionEndpoint } from './types/collection'
+import { IntegrationEndpoint } from './types/integration'
+import { OrdersEndpoint } from './types/order'
+import { GatewaysEndpoint } from './types/gateway'
+import { CustomersEndpoint } from './types/customer'
+import { InventoryEndpoint } from './types/inventory'
+import { JobEndpoint } from './types/job'
+import { FileEndpoint } from './types/file'
+import { FlowEndpoint } from './types/flow'
+import { FieldsEndpoint } from './types/field'
+import { AddressesEndpoint } from './types/address'
+import { TransactionEndpoint } from './types/transaction'
+import { SettingsEndpoint } from './types/settings'
+import { PromotionsEndpoint } from './types/promotions'
+import { VariationsEndpoint } from './types/variations'
+import { AuthenticationSettingsEndpoint } from './types/authentication-settings'
+import { OidcProfileEndpoint } from './types/oidc-profile'
+import { AuthenticationRealmEndpoint } from './types/authentication-realm'
+import { CartEndpoint } from './types/cart'
```

Comment:
I just updated the imports to only use what is being consumed, they are all getting exported below, so just seems neater to me. :)